### ISSUE TITLE
Hi! different functions of feature maps.

### ISSUE THREAD

**Issue by wqz960** [utkuozbulak/pytorch-cnn-visualizations#80]
Great work! Can you tell me the function of different feature maps, what function are they used to tell us? Thank you !

**Comment by utkuozbulak** [comment]
Hello,

You can read relevant papers to get informed on this topic (papers are linked in the README).

**Comment by wqz960** [comment]
Hi! the guided backprop function means that during BP, the gradient = 1 where gradient > 0 and the gradient = 0 where gradient < 0 through RELU activation? Is it True? Thank you!
